docs/MigrationGuide.md

```diff
@@ -17,7 +17,8 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   `helper-utils` to `default-helpers`, separated `MathUtils` from
   `components-utils` to `math-utils`, removed dependencies from `helper-utils`
 * [#3420](https://github.com/TouK/nussknacker/pull/3420) `DeploymentManagerProvider.typeSpecificInitialData` takes deploymentConfig `Config` now
-* [#3493](https://github.com/TouK/nussknacker/pull/3493) Added method `DeploymentManagerProvider.additionalPropertiesConfig`, which allows to configure additional scenario properties programmatically.              
+* [#3493](https://github.com/TouK/nussknacker/pull/3493) Added method `DeploymentManagerProvider.additionalPropertiesConfig`, which allows to configure additional scenario properties programmatically.
+* [#3505](https://github.com/TouK/nussknacker/pull/3505) Bump node version to 16.5.1
```

Comment:
I think it doesn't affect users/deployments of Nussknacker?

Reply:
Ok, this line was removed.